content/04.results.md

```diff
@@ -3,11 +3,66 @@
 ## Random Sampling of Generative Model
 place the grid aurocs here for generative model
 
-## Discriminator Model Builds Off Generative Model
-place the grid of aurocs here for discriminator model
+### Discriminator Model Builds Off Generative Model
+In this framework we used a generative model trained over label functions to produce probabilistic training labels for each sentence.
+Then we train a discriminative model, which has full access to a representation of the text of the sentence, to predict the generated labels.
+The discriminative model is a convolutional neural network trained over word embeddings.
+We report the results of the discriminative model using area under the precision recall curve (AUPR) (Figure {@fig:discriminative_model_auprc}) and area under the receiver operating curve (AUROC) (Figure {@fig:discriminative_model_auroc}).    
+Generally, the discriminative model under-performs compared to the generative model.
+In the top left corner of figures {@fig:discriminative_model_auroc} and {@fig:discriminative_model_auprc} the discriminative model underperforms the generative model when only using distant supervision label functions.
+As the number of label functions increase the discriminative model starts to have similar performance to the generative model.
+Compound-treats-Disease (CtD) starts off with the opposite trend.
+The discriminative model outcompetes the generative model; however, as more label functions are added performance starts to decline.
+For the Compound-binds-Gene (CbG) edge type, the discriminative model consistently underperforms the generative model.
+Gene-interacts-Gene (GiG) has a similar pattern to DaG.
+The discriminative model starts off underperforming the generative model.
+As soon as more label functions are incorporated performance improves to be on par with the generative model.  
```

Comment:
```suggestion
```

Deleting a ton of this. You don't need to describe every panel of the figure (that'd take a thousand words, since an image is worth a thousand words 😉 ). You need to give the main take-home points and cite the figure for support.